### DISCUSSION TITLE
[Support] 如何获取某个类的所有方法的元信息，包括请求参数，返回类型等

### DISCUSSION THREAD

**Discussion by 3454186042** [midwayjs/midway#4134] [Q&A]
### Describe the problem（描述问题）

通过listModule, 我获取到了所有绑定到某类型装饰器的 class， 接下去我想获取到该class下所有的请求方法的元信息：path, method, 描述， 请求参数&类型，返回类型， 请问该怎么获取？

我从getClassMetadata(WEB_ROUTER_KEY, xxxController)可以获取到path、method、description， 但是函数的请求参数&类型，返回类型不知如何获取

### Midway Versions（Midway 版本）

midway-version@1.4.0

**Comment by czy88840616** [comment]
有 https://github.com/midwayjs/midway/blob/main/packages/core/src/decorator/decoratorManager.ts#L934 和 https://github.com/midwayjs/midway/blob/main/packages/core/src/decorator/decoratorManager.ts#L923 这两个方法，都是代理了 reflect的属性。
